source/Preferences.cpp

```diff
@@ -51,6 +51,9 @@ namespace {
 
 	const vector<string> BOARDING_SETTINGS = {"proximity", "value", "mixed"};
 	int boardingIndex = 0;
+
+	const vector<string> ALERT_INDICATOR_SETTING = {"off", "audio", "visual", "both"};
+	int alertIndicatorIndex = 3;
```

Comment:
should it really start with both?

Reply:
Yes. The current default is to have the audible siren on, I don't see why we wouldn't also have this indicator on.

Reply:
ok